server/src/internalClusterTest/java/org/opensearch/remotestore/RemoteRestoreSnapshotIT.java

```diff
@@ -327,16 +329,22 @@ public void testRemoteStoreCustomDataOnIndexCreationAndRestore() {
         validatePathType(indexName2, PathType.HASHED_PREFIX, PathHashAlgorithm.FNV_1A);
```

Comment:
as this value will be in index metadata, do we need to override it during restore based on cluster defaults? otherwise i think it will be picked from snapshot metadata.

Reply:
The index metadata needs to be overridden to ensure that the restored index uses the latest path strategy. This has been done currently already.